### ISSUE TITLE
maybe missing best feature of Airbnb?

### ISSUE THREAD

**Issue by cjavdev** [ajberk/SurpriseBNB#2]
I like the idea, i feel that the key awesome UI feature of AirBNB is moving the map around and seeing the listings update tho?

**Comment by jeffnv** [comment]
Yeah I agree with CJ on this one. Without the map it's just not as cool
